lib/stream_data.ex

```diff
@@ -136,6 +157,11 @@ defmodule StreamData do
   end
 
   # TODO: should this be exposed?
+  # QUESTION: In Elixir, we would call this function filter_map,
+  # and have it return {:cont, value} | :skip or {:ok, value} | :error.
+  # Maybe we should standardize this return type and make this public?
+  # I would also likely call the underlying rose tree operation filter_map.
```

Comment:
Sorry, I did not meant to call it `filter_map`, I am mostly proposing to unify the function return types (and maybe call the lazy tree one filter_map).

> About making it public, we can but maybe we can wait, I don't know. Preferences?

Property calls it and it would be nice if ExUnit did not depend on internal APIs.